### DISCUSSION TITLE
[Show and Tell] RetroPGF3 Project Eligibility Checker and Discovery

### DISCUSSION THREAD

**Discussion by Chomtana** [ethereum-optimism/developers#27] [Show and tell]
### Description

For those who want to check for your RetroPGF 3 project eligibility, you can check it via RetroList now!
https://retrolist.app/

![image](https://github.com/ethereum-optimism/developers/assets/4103490/bca01f75-d3b7-43ab-8ecd-cd34c203b02e)

Put your project name in the search box and check if your project is eligible for RetroPGF3.

Soon, we will support Rubric-based list creation.

**Comment by sbvegan** [comment]
@Chomtana thanks for sharing! This has a very clean UI! What are the, "Removed" listings? Applications that you don't think fit the RPGF criteria?

**Comment by Chomtana** [comment]
Hi @sbvegan the listing status is based on the official RetroPGF 3 reviewing sheet

https://docs.google.com/spreadsheets/d/1o1AZNEPiESrluvTEUBJgFqFWwI6G9xihGy3VL1ecNaQ/edit#gid=1862166755

Currently, there are 4 statuses
- Eligible: The project is eligible for RetroPGF 3.
- Removed: The project is removed from RetroPGF 3 with a chance to appeal on 2-3 Nov 2023.
- Reviewing: The project is under review because it is being reported for violating a rule.
- Pending Approval: The project will be approved on 1 Nov 2023 if there aren't any further rule violation reports.

**Comment by Chomtana** [comment]
![image](https://github.com/ethereum-optimism/developers/assets/4103490/391aee05-4ffc-4922-b767-3dc0dffca210)

Added a status filter now

**Comment by Chomtana** [comment]
RetroList is now supported for viewing the final review results along with the reason!
![image](https://github.com/ethereum-optimism/developers/assets/4103490/b8ebc57e-45df-41ce-b5f0-a1b07a1100f1)

**Comment by sbvegan** [comment]
@Chomtana this looks great! Is this a solo project or a governance mission you're taking on?

**Reply by Chomtana** [reply]
I am doing it with other two friends. A UI/UX designer and a frontend dev.

**Reply by Chomtana** [reply]
There is no governance mission. I have submitted RFP once but it is rejected!

**Reply by sbvegan** [reply]
Ah I just checked that out, it looks like Agora is doing the one for voting. Nonetheless, this is a great looking project!

**Reply by 173946** [reply]
Good job
